pkg/services/apiserver/aggregator/availableController.go

```diff
@@ -225,7 +228,7 @@ func (c *AvailableConditionController) sync(key string) error {
 						_ = resp.Body.Close()
 						// we should always been in the 200s or 300s
 						if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
-							errCh <- fmt.Errorf("bad status from %v: %v", discoveryURL, resp.StatusCode)
+							errCh <- fmt.Errorf("bad status from %v: %d", discoveryURL, resp.StatusCode)
```

Comment:
A minor change in upstream (remote available conditon controller). Other changes are only string changes changing the name of the controller. Since, we still retain the logic for local services here, I have decided to keep the name the same "AvailableConditonController".